### PR TITLE
chore: change registry.civil.co to civil.co in support of switchover

### PR DESCRIPTION
**Pull request by nickreynolds**

### CODE REVIEW

File: packages/utils/src/urlConstants.ts [hunk 1]
```diff
@@ -2,10 +2,10 @@ const FAQ_BASE_URL = "https://help.civil.co";
...
-  REGISTRY: "https://registry.civil.co/",
-  DASHBOARD: "https://registry.civil.co/dashboard",
-  CONSTITUTION: "https://civil.co/constitution",
+  MARKETING_SITE: "https://learn.civil.co",
+  REGISTRY: "https://civil.co/",
+  DASHBOARD: "https://civil.co/dashboard",
+  CONSTITUTION: "https://learn.civil.co/constitution",
   FOUNDATION: "https://civilfound.org",
   COUNCIL: "https://civilfound.org/#civil-council",
   APPLY: "https://civil.co/join-as-a-newsroom",
```
Comment (tobek) [inline packages/utils/src/urlConstants.ts]: Ditto. I see https://civil.co/join-as-a-newsroom/ exists but https://learn.civil.co/join-as-a-newsroom/ does not exist though, not sure what the plan is for that page.

File: packages/dapp/src/helpers/links.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
   // Marketing site  / Foundation
-  MARKETING_SITE: "https://civil.co",
-  REGISTRY: "https://registry.civil.co/",
-  CONSTITUTION: "https://civil.co/constitution",
+  MARKETING_SITE: "https://learn.civil.co",
+  REGISTRY: "https://civil.co/",
+  CONSTITUTION: "https://learn.civil.co/constitution",
   FOUNDATION: "https://civilfound.org",
   COUNCIL: "https://civilfound.org/#civil-council",
   APPLY: "https://civil.co/join-as-a-newsroom",
```
[the remark this answers is not in the document]
Comment (nickreynolds) [reply packages/dapp/src/helpers/links.ts]: just gonna delete these. doesn't seem like they're being used anywhere

[the remark this answers is not in the document]
Comment (nickreynolds) [reply packages/dapp/src/helpers/links.ts]: actually the member one is still used

**Comment by dankins** [comment]
:tada: This PR is included in version 4.9.5 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**